src/Microsoft.VisualStudio.AppDesigner/ApplicationDesigner/ApplicationDesignerEditorFactory.vb

```diff
@@ -15,18 +15,20 @@ Namespace Microsoft.VisualStudio.Editors.ApplicationDesigner
     ';ApplicationDesignerEditorFactory
     '
     'Remarks:
-    '   The editor factory for the resource editor.  The job of this class is
-    '   simply to create a new resource editor designer when requested by the
+    '   The editor factory for the application designer. The job of this class is
+    '   simply to create a new application designer when requested by the
     '   shell.
     '**************************************************************************
     <CLSCompliant(False),
-    Guid("04b8ab82-a572-4fef-95ce-5222444b6b64"),
+    Guid(ApplicationDesignerEditorFactory.EditorGuidString),
     ProvideView(LogicalView.Designer, "Design")>
     Public NotInheritable Class ApplicationDesignerEditorFactory
         Implements IVsEditorFactory
 
+        Friend Const EditorGuidString = "04b8ab82-a572-4fef-95ce-5222444b6b64"
+
         'The all important GUIDs 
-        Private Shared ReadOnly s_editorGuid As New Guid("{04b8ab82-a572-4fef-95ce-5222444b6b64}")
+        Private Shared ReadOnly s_editorGuid As New Guid(EditorGuidString)
```

Comment:
Just to confirm, `New Guid(...)` will correctly parse the string without the curly braces at the beginning and end?

Reply:
👍 yes, it handles both.